src/H5Fmodule.h

```diff
@@ -1041,6 +1041,50 @@
  * additional variable settings associated with the Split driver, there is no H5Pget_fapl_split
  * function.
  *
+ * \subsubsection subsubsec_file_alternate_drivers_ros3 The ROS3 Driver
+ * The ROS3 (read-only S3) driver is used to enable read-only access to HDF5 files which are stored
```

Comment:
I added a user's guide entry for the VFD, but as the entries for the other VFDs were brief I didn't add too many details here. Most information about the VFD is in its header file currently. But I think we should revise these entries into separate pages or something similar so they can serve as the main location for information about a particular VFD. That or maybe free-form doxygen like this should be put into each VFD's header file.

Reply:
The H5?module.h files were convenient (initially) for User Guide entries as they wouldn't be confused/interfere with reference Manuals entries and some UG entries were extensive. Anything should be able to be reworked/relocated as necessary.